demos/browser/server.js

```diff
@@ -199,6 +199,8 @@ function serve(host = '127.0.0.1:8080') {
         await client.deleteMeeting({
           MeetingId: meetingTable[requestUrl.query.title].Meeting.MeetingId,
         }).promise();
+        //Remove the meeting from table(so that a new meeting can be created with the same name with in a session)
+        delete meetingTable[requestUrl.query.title];
```

Comment:
This change only apply for local dev. How about our severless demo? For that one, we currently set ttl to 24h https://github.com/aws/amazon-chime-sdk-js/blob/main/demos/serverless/src/handlers.js#L544

Reply:
I thought of updating the serverless demo too, but wasn't sure if there was a reason for setting that 24 ttl. I have the code for that as well, but did not push it. I can push that as well if you are okay with that. That way we delete the entry from dynamo if meeting ends and if all participants leave without ending the meeting, the entry would be deleted after 24 hours. Let me know.

Reply:
@ltrung - committed the handler.js code for your reference. One thing to note is there is a slight delay in UI from when the end meeting button is clicked to taking back the new meeting landing page that is introduced by the dynamo delete call. I don't think it's a big deal, but wanted to make a note of it.